bin/setup

```diff
@@ -28,6 +28,16 @@ if [ -z "$CI" ]; then
   rm -rf node_modules
   rm -rf deps _build
   asdf install
+else
+  [[ ! -d ~/.asdf ]] && git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.4.1
+  . $HOME/.asdf/asdf.sh
+  echo -e '\n. $HOME/.asdf/asdf.sh' >> ~/.bashrc
+  if ! grep -E nodejs <(asdf plugin-list) > /dev/null; then
+    asdf plugin-add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+    bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
```

Comment:
I don't think we should do this in a setup script. This is added GPG keys to someone's keyring which doesn't feel like the kind of thing we should do automatically, especially in a private project.

Maybe we should put out that they need to do this?

Reply:
Thanks. Ya, that's not cool. I was thinking in the interest of getting the project ready in a single command, but you're right, I wouldn't want some random project telling my computer to trust others' signatures.

Reply:
On second thought, this section of code is about handling CI setup. In that case, I don't think this matters.

Reply:
Ah, sorry I misread that. Rock on.